src/taipy/core/_version/_version_manager.py

```diff
@@ -32,7 +33,7 @@ class _VersionManager(_Manager[_Version]):
 
     _DEFAULT_VERSION = __LATEST_VERSION
 
-    _repository = _VersionRepositoryFactory._build_repository()  # type: ignore
+    _repository: _VersionFSRepository
```

Comment:
should this be _AbstractRepository?

Reply:
No, Abstract repository does not have specific methods used on Version. My idea is to make this type hint a Union[_VersionFSRepository, _VersionSQLRepository] when the refactoring is done, to avoid mypy issues.